README.md

```diff
@@ -4,7 +4,8 @@
 <img src="./src/assets/logo.svg" alt="Logo" width="150px">
   <h2><b>Crypto Angels</b></h2>
 </div>
-   <h4><b>Readme</b></h4>
+
```

Comment:
- [ ] In your test files, I only can see `snapshot` tests. But please also implement the unit. Also, you can perform a mocking data fetching and Redux Store connection.

Reply:
Hello @Rustamxon7 

I have made unit tests as well as per your remarks. Now, again asking for a review.

Thank you